ADOPTERS.md

```diff
@@ -81,4 +81,5 @@ This list is sorted in the order that organizations were added to it.
 | [Variphy](https://www.variphy.com/) | [info@variphy.com](mailto:info@variphy.com) ([@dmvariphy](https://github.com/dmvariphy) [@nick-variphy](https://github.com/nick-variphy) [@dfalter-variphy](https://github.com/dfalter-variphy) [@zach-variphy](https://github.com/zach-variphy)) | We use Crossplane (via [Upbound Cloud](https://www.upbound.io/)) to manage our development and production infrastructure via GitOps.  Crossplane also allows us to provide custom APIs for production Variphy applications to dynamically manage external resources, such as [Confluent Cloud](https://www.confluent.io/) Kafka topics.  |
 | [OneUptime](https://oneuptime.com) | @simlarsen | Builds production and developer environments that power the OneUptime Platform. |
 | [Xata](https://xata.io) | [@mattfield](https://github.com/mattfield) [@paulaguijarro](https://github.com/paulaguijarro) | Crossplane manages the dev, staging, and production RDS Aurora PostgreSQL clusters for our [Dedicated Clusters](https://xata.io/blog/postgres-dedicated-clusters) offering, along with Flux Kustomizations and other resources that provision cells of internal [Xata](https://xata.io) services. |
-| [AlphaSense](https://www.alpha-sense.com/) | @abhihendre | Engineering teams at [AlphaSense](https://www.alpha-sense.com/) leverage Crossplane APIs, abstracted by a set of Helm charts and Compositions curated by our Platform Teams, to seamlessly provision cloud services across three major clouds, including our production environment.|
\ No newline at end of file
+| [AlphaSense](https://www.alpha-sense.com/) | @abhihendre | Engineering teams at [AlphaSense](https://www.alpha-sense.com/) leverage Crossplane APIs, abstracted by a set of Helm charts and Compositions curated by our Platform Teams, to seamlessly provision cloud services across three major clouds, including our production environment.|
+| [CloudScript](https://www.cloudscript.com.br/) | @xcloudscript | [CloudScript](https://www.cloudscript.com.br/) engineers have been using Crossplane since 2022 creating customized Compositions for the implementation of our engineering platform, basically automating the creation of Kubernetes environments on AWS, GCP and Azure ( coming soon).|
```

Comment:
@xcloudscript this is great!! is this also running in production environments for you all? if so, would you mind updating this text with something like "...engineering platform in production environments, basically automating..."

Also tiny nit: there's an extra whitespace after the opening parenthesis in `( coming soon)` if you feel like cleaning that up too.